flatpak-package.patch

```diff
@@ -87,6 +87,7 @@ index 91fea1a84..04ca0109b 100755
 -		$(DESTDIR)/usr/share/applications/
 -	install -d $(DESTDIR)/usr/share/menu/
 +		$(DESTDIR)/app/share/applications/
++	sed -i 's,/usr,/app,' $(DESTDIR)/app/share/applications/pd-extended.desktop
```

Comment:
I don't see why this is necessary. The Exec line is fixed in the manifest already.